kafka-streams/src/main/java/io/micronaut/configuration/kafka/streams/KafkaStreamsFactory.java

```diff
@@ -39,6 +41,7 @@
  * @since 1.0
  */
 @Factory
+@Requires(property = AbstractKafkaConfiguration.PREFIX + "." + AbstractKafkaConfiguration.ENABLED_CONFIG, value = "true", defaultValue = "true")
```

Comment:
Wouldn't this be better?

```suggestion
@Requires(property = AbstractKafkaConfiguration.PREFIX + "." + AbstractKafkaConfiguration.ENABLED_CONFIG, value = "true", defaultValue = AbstractKafkaConfiguration.DEFAULT_ENABLED)
```